src/presets/firebase/preset.ts

```diff
@@ -88,7 +88,7 @@ const firebaseAppHosting = defineNitroPreset(
           stringifyYAML({
             version: "v1",
             runConfig: {
-              runCommand: "node .output/server/index.mjs",
+              runCommand: `node ${join(nitro.options.output.serverDir, "index.mjs")}`,
```

Comment:
We should generate it relative to `options.rootDir` or `options.workspaceDir`, currently it leads to an absolute path like this:

```
 runCommand: node /Users/pooya/Code/nitro/playground/.output/server/index.mjs
```

Reply:
19c674a09485cf9d8e241fd5fd78b45c4b148b7e